src/lib/utils/parseStringToList.ts

```diff
@@ -0,0 +1,10 @@
+export function parseStringToList(links: unknown): string[] {
+	if (typeof links !== "string") {
+		return [];
```

Comment:
shouldn't it explicitly be `links: stringp[]` rather than `links: unknown` and implictly returning `emtoty array []` when it is not array?

Reply:
it has to be `unknown` for the input since we use it in a [preprocessing step](https://github.com/huggingface/chat-ui/blob/bf2cb52f3ceb95d9ded418dcd7e17504a4dda4e4/src/routes/settings/(nav)/assistants/%5BassistantId%5D/edit/%2Bpage.server.ts#L24-L25) with zod, but yeah it could throw an error if it's not a string instead of returning []